frontend/src/components/TopBar.tsx

```diff
@@ -21,18 +21,15 @@ import {useLogger} from '../logger/LoggerProvider'
 import {NavigateFunction, useNavigate} from 'react-router-dom'
 
 function regions(selected: any) {
```

Comment:
suggestion: Let's add a test to this function, since we are adding some business logic

Reply:
Will add in next revision :)